generators/csharp/codegen/src/asIs/JsonConfiguration.Template.cs

```diff
@@ -94,6 +95,28 @@ internal static string SerializeAsString<T>(T obj)
         return json.Trim('"');
     }
 
+    internal static string SerializeWithAdditionalProperties<T>(T obj, Dictionary<string, object>? additionalProperties = null)
```

Comment:
I'm in favor of making `additionalProperties` an object.
Since we don't have to worry about symmetric request/response type here.
That way, users can make anonymous objects or even create their own classes with JSON attributes to get it as they wish.
The only downside is, that someone could pass in primitive types likes int's which doesn't make sense, so I'd just check that when you serialize the object, the JSON node type is object.